rs/sns/governance/src/proposal.rs

```diff
@@ -1826,6 +1832,120 @@ fn validate_and_render_advance_sns_target_version_proposal(
     ))
 }
 
+fn topic_to_str(topic: &Topic) -> &'static str {
+    match topic {
+        Topic::Unspecified => "Unspecified",
+        Topic::DaoCommunitySettings => "DaoCommunitySettings",
+        Topic::SnsFrameworkManagement => "SnsFrameworkManagement",
+        Topic::DappCanisterManagement => "DappCanisterManagement",
+        Topic::ApplicationBusinessLogic => "ApplicationBusinessLogic",
+        Topic::Governance => "Governance",
+        Topic::TreasuryAssetManagement => "TreasuryAssetManagement",
+        Topic::CriticalDappOperations => "CriticalDappOperations",
+    }
+}
```

Comment:
I think Topic already has [this functionality][fun]?

[fun]: https://sourcegraph.com/github.com/dfinity/ic@85184bf1284b2535763b72f7ffb9650528ade9af/-/blob/rs/sns/governance/src/gen/ic_sns_governance.pb.v1.rs?L4230

Reply:
This function produces string that match the Candid values, which I think is better for the end users. Cf. `TOPIC_APPLICATION_BUSINESS_LOGIC` and `ApplicationBusinessLogic`.